### ISSUE TITLE
google maps does not load properly

### ISSUE THREAD

**Issue by shahmanash** [AtlasOfLivingAustralia/collectory-plugin#87]
update google maps api js

```
<g:if test="${grailsApplication.config.google.apikey}">
        <script async defer src="https://maps.googleapis.com/maps/api/js?key=${grailsApplication.config.google.apikey}" type="text/javascript"></script>
</g:if>
```

`<script type="text/javascript" src="http://maps.google.com/maps/api/js?v=3.3&sensor=false"></script>
`

**Comment by djtfmartin** [comment]
thanks @shahmanash 

would you be able to send a patch  or fork and pull request ?

**Comment by djtfmartin** [comment]
i think this is fixed now in 1.6
